components/movementsensor/gpsrtk/model_test.go

```diff
@@ -29,7 +28,7 @@ func TestGPSModels(t *testing.T) {
 			}
 					]
 		}`
-		_, err := config.FromReader(ctx, "", strings.NewReader(cfg1), logger)
```

Comment:
this function name may not make sense anymore because we are passing in raw bytes rather than a reader for those bytes. Wanted to minimize big changes in this PR, so left out the naming change for now, but can make fromBytes or something instead.

Reply:
Yeah should really be config.FromBytes. Ok to refactor later, can you create a ticket in the Modular Registry epic?

Reply:
changed as it wasn't difficult to do, and im here now 🤷🏼